tests/utils/exec/rosacli/break_glass_credential_service.go

```diff
@@ -62,148 +70,222 @@ func NewBreakGlassCredentialService(client *Client) BreakGlassCredentialService
 
-func (b *breakglasscredentialService) WaitForBreakGlassCredentialToStatus(clusterID string, status string, userName string) wait.ConditionFunc {
+func (b *breakglasscredentialService) WaitForStatus(clusterID string, status string, userName string) wait.ConditionFunc {
 	return func() (bool, error) {
-		breakGlassCredList, err := b.ListBreakGlassCredentialsAndReflect(clusterID)
+		breakGlassCredList, err := b.List().Parameters(clusterID).ToStruct()
 		if err != nil {
 			return false, err
 		}
-		_, breakGlassCredential := breakGlassCredList.IsPresent(userName)
+		_, breakGlassCredential := breakGlassCredList.(BreakGlassCredentialList).IsPresent(userName)
 		Logger.Infof("The status for break-glass-credential %s is %s\n", breakGlassCredential.ID, breakGlassCredential.Status)
 		return breakGlassCredential.Status == status, err
 	}
 }
 
-// Help for Create BreakGlassCredential
-func (b *breakglasscredentialService) RetrieveHelpForCreate() (output bytes.Buffer, err error) {
-	return b.client.Runner.Cmd("create", "break-glass-credential").CmdFlags("-h").Run()
-}
+// // Help for Create BreakGlassCredential
+// func (b *breakglasscredentialService) RetrieveHelpForCreate() (output bytes.Buffer, err error) {
+// 	return b.client.Runner.Cmd("create", "break-glass-credential").CmdFlags("-h").Run()
+// }
 
-// Help for List BreakGlassCredential
-func (b *breakglasscredentialService) RetrieveHelpForList() (output bytes.Buffer, err error) {
-	return b.client.Runner.Cmd("list", "break-glass-credential").CmdFlags("-h").Run()
-}
+// // Help for List BreakGlassCredential
+// func (b *breakglasscredentialService) RetrieveHelpForList() (output bytes.Buffer, err error) {
+// 	return b.client.Runner.Cmd("list", "break-glass-credential").CmdFlags("-h").Run()
+// }
 
-// Help for Describe BreakGlassCredential
-func (b *breakglasscredentialService) RetrieveHelpForDescribe() (output bytes.Buffer, err error) {
-	return b.client.Runner.Cmd("describe", "break-glass-credential").CmdFlags("-h").Run()
-}
+// // Help for Describe BreakGlassCredential
+// func (b *breakglasscredentialService) RetrieveHelpForDescribe() (output bytes.Buffer, err error) {
+// 	return b.client.Runner.Cmd("describe", "break-glass-credential").CmdFlags("-h").Run()
+// }
 
-// Help for Delete BreakGlassCredential
-func (b *breakglasscredentialService) RetrieveHelpForDelete() (output bytes.Buffer, err error) {
-	return b.client.Runner.Cmd("revoke", "break-glass-credential").CmdFlags("-h").Run()
-}
+// // Help for Delete BreakGlassCredential
+// func (b *breakglasscredentialService) RetrieveHelpForDelete() (output bytes.Buffer, err error) {
+// 	return b.client.Runner.Cmd("revoke", "break-glass-credential").CmdFlags("-h").Run()
+// }
 
 func (b *breakglasscredentialService) CleanResources(clusterID string) (errors []error) {
 	if b.created[clusterID] {
 		Logger.Infof("Remove remaining break-glass-credentials")
-		_, err := b.DeleteBreakGlassCredential(clusterID)
+		_, err := b.Revoke().Help().Run()
```

Comment:
This is an incorrect replacement. It should be `b.Revoke().Parameters(clusterID).Run()`